src/app/forms/templates/test-records/create-master.template.ts

```diff
@@ -145,6 +146,17 @@ export const contingencyTestTemplates: Record<VehicleTypes, Record<string, Recor
       defects: defectsHiddenSection,
       customDefects: CustomDefectsHiddenSection,
       reasonForCreation: reasonForCreationSection
+    },
+    testTypesDeskBasedGroup3: {
+      required: CreateRequiredSection,
+      vehicle: DeskBasedVehicleSectionDefaultPsvHgv,
```

Comment:
`DeskBasedVehicleSectionDefaultPsvHgv` only contains VIN, VRM, Country of registration & EU Vehicle Category. Whereas you also need Odometer reading & Odometer units.

I'd make a new template for this; I couldn't find one that conforms to your requirements.

Reply:
DeskBasedVehicleSectionDefaultPsvHgv does contain odometer reading and units